sdv/tabular/base.py

```diff
@@ -525,13 +503,23 @@ def sample(self, num_rows, randomize_samples=True, max_tries_per_batch=100, batc
         batch_size = min(batch_size, num_rows) if batch_size else num_rows
 
         try:
-            with tqdm.tqdm(total=num_rows) as progress_bar:
-                progress_bar.set_description('Sampling rows')
+            if show_progress_bar:
+                with tqdm.tqdm(total=num_rows) as progress_bar:
+                    progress_bar.set_description('Sampling rows')
+                    sampled = self._sample_in_batches(
+                        num_rows=num_rows,
+                        batch_size=batch_size,
+                        max_tries_per_batch=max_tries_per_batch,
+                        progress_bar=progress_bar,
+                        output_file_path=output_file_path
+                    )
+
+            else:
                 sampled = self._sample_in_batches(
                     num_rows=num_rows,
                     batch_size=batch_size,
                     max_tries_per_batch=max_tries_per_batch,
-                    progress_bar=progress_bar,
+                    progress_bar=None,
```

Comment:
we could remove the `if - else` by only adding `tqdm.tqdm(total=num_rows, disable=not show_progress_bar)` (When `True` is not being shown, when `False` is being shown, defaults to `False`).